src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-advanced-dashboarding.mdx

```diff
@@ -0,0 +1,471 @@
+
+In this tutorial, you'll learn how to advance your dashbaording by using faceting by case, advanced aggregation functions, the `EXTRAPOLATE` keyword, filtering aggregation functions, and overriding values. Specifically, you'll learn how to use:
+
+* Advanced aggregation functions like `filter()`, `apdex()`, `rate()funnel()`, `histogram()`.
+* The `EXTRAPOLATE` clauses.
+* `FACET CASES()`, including how to use attribute and group matching values.
+* `filter()` to combine Event Types.
+* Overriding values, when necessary.
+
+You may not use these functions and features on every single dashboard, but they will certainly come in handy when tackling specific problems and requirements. Let's get started!
+
+### Use advanced aggregators 
+<Steps>
+    <Step>
+#### Calculating Rate
+Let's start with the `rate()` function. It allows you to visualize the frequency of events over time, which helps when you want to understand the frequency of events in small periods of time within larger time windows.
+
+In the example below, you can see the average frequency of requests every 5 minutes for the last hour compared to the previous hour's 5-minute average frequency. Notice the query uses `SINCE 1 hour ago`, this is the overall time window in which you're calculating the rate.
+
+<SideBySide>
+    <Side>
+```sql
+SELECT rate(count(*), 5 minutes) 
+FROM Public_APICall 
+SINCE 1 hour ago 
+COMPARE WITH 1 hour ago
+```
+    </Side>
+    <Side>
+<img
+    title="Calculate range"
+    alt="A screenshot of a query using Since and Compare to capture data within a time range"
+    src={queriesnrql2tutorial1}
+/>
+    </Side>
+</SideBySide>
+
+You can use `rate()` to calculate requests per minute or requests per second by setting the time interval to either 1 minute or 1 second.
+
+    </Step>
+    <Step>
+#### Funnel Charts
+Funnel charts track the occurrence of an attribute value across multiple records, and many people use them to understand end-user behavior. You might often use them to visualize how successfully users' progress through defined paths, and especially when using custom attributes.
+
+You can use the `funnel()` aggregator function to visualize how many users visit a specific page before moving on to another page. The first parameter is the identifying attribute for the unique entries you're counting. In this case, New Relic Browser assigns and retains a `session ID` attribute for each user on your site (subject to cookies being enabled). You can also set your own session ID using custom attributes.
```

Comment:
See product caps 'New Relic Browser.' Not sure that's the style we use, but confirm